src/node_options.cc

```diff
@@ -63,6 +63,15 @@ void PerProcessOptions::CheckOptions(std::vector<std::string>* errors) {
                       "used, not both");
   }
 #endif
+
+// #ifdef NODE_REPORT
```

Comment:
Do I need to move this check in per-process option checks? Do we still need a dependency on `--experimental-report` flag being set.

Reply:
yes, I'd say so.

Reply:
> Do we still need a dependency on `--experimental-report` flag being set.

I think that gets a bit tricky here, so I’m fine with omitting it.